src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classnames from 'classnames';
+import { User } from '../../type/user';
+import { Todo } from '../../type/todo';
+
+type Props = {
+  todo: Todo;
+  findUser: (id: number) => User | undefined;
```

Comment:
why do you need to pass that function here as the prop if you just can add it in the component as the simple function